server/src/diagnostics.ts

```diff
@@ -436,6 +437,31 @@ export class Diagnostics {
     return range
   }
 
+  private packageNameRangeFromImportDeclaration(textDocument: TextDocument, packageName: string, loc: Acorn.SourceLocation): Range {
```

Comment:
Hey @marcoroth, I decided to just update the package name.

In order to do this, I had to create this additional helper method that would find the package name substring within the import statement and update it.

Reply:
I'm not sure I follow 😞.

Reply:
No worries, we will get there! 

Let me see if I can sketch up the basic idea so I can show what I mean.

Reply:
Hey @nachiket87, I pushed up https://github.com/nachiket87/stimulus-lsp/commit/ceee1c531f18dcc98250dc85faa6fffa8af6fd3a to your branch, which sketeches out the basic idea I was talking about.

This approach uses the `data` that is attached to the diagnostic to carry the `importSourceRange`, which is only the range of the actual string of literal between the quotes.

I also added a `updateImportSource` in `Commands`, which gets called when the LSP sees a `stimulus.import.source.update` command. It feels cleaner to have this as a separate command, opposed to "re-use" the existing controller update command, which is mostly meant to replace controller occurrences in HTML documents.

Let me know if that makes sense or if you have any questions!

Reply:
@marcoroth Yes, this is great! I see what you did.  I was on the same path but I couldn't find the loc of just the `"stimulus"` import. Specifically this one you wrote:

`const importSourceRange = this.rangeFromLoc(textDocument, importDeclaration.node.source.loc)`

 Does this complete the ticket? I tested out your changes and it works.

Reply:
Yeah, I think this might do it! Sorry for "snagging" this away from you! Let me know if you want to pair one another one!

Reply:
No problem @marcoroth I am currently investigating https://github.com/marcoroth/stimulus-lsp/issues/162